src/test/java/org/jboss/xavier/integrations/migrationanalytics/business/VMWorkloadInventoryCalculatorTest.java

```diff
@@ -93,6 +93,9 @@ public void calculate_jsonV1_0_0_Given_ShouldReturnCalculatedValues() throws IOE
         assertThat(modelList.stream().filter(e -> e.getGuestOSFullName().equalsIgnoreCase("CentOS 7 (64-bit)")).count()).isEqualTo(1);
         assertThat(modelList.stream().filter(e -> e.getGuestOSFullName().equalsIgnoreCase("Red Hat Enterprise Linux Server release 7.6 (Maipo)")).count()).isEqualTo(6);
         assertThat(modelList.stream().filter(e -> e.getDiskSpace() == (17980588032L)).count()).isEqualTo(1);
+        assertThat(modelList.stream().filter(e -> e.getHasCpuHotAdd() != null).count()).isEqualTo(1);
+        assertThat(modelList.stream().filter(e -> e.getHasCpuHotRemove() != null).count()).isEqualTo(1);
+        assertThat(modelList.stream().filter(e -> e.getHasMemoryHotAdd() != null).count()).isEqualTo(1);
```

Comment:
what do you think about testing that a concrete VM has the 3 fields != null and another one having all as null because you dont set any of the 3 fields in the VM ? just a suggestion.... :)

Reply:
I've added an extra expectedValue tests which tests one of the VMs with all these fields unset, and altered the other one so all three != null